### DISCUSSION TITLE
Azure AD auth

### DISCUSSION THREAD

**Discussion by mani-zur** [chatwoot/chatwoot#8495] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
In previous version Google Oauth was added. I think azure AD will be usefull too.

**Describe the solution you'd like**
I would like to be able to use Azure Ad to log into Chatwoot.

**Additional context**
[https://learn.microsoft.com/en-gb/azure/active-directory/fundamentals/auth-oauth2](https://learn.microsoft.com/en-gb/azure/active-directory/fundamentals/auth-oauth2)

**Comment by pranavrajs** [comment]
Moving this to feature discussions as we won't be prioritizing this any time soon.
